x/did/types/did.go

```diff
@@ -590,23 +591,63 @@ func (didDoc DidDocument) HasRelationship(
 	for _, vm := range didDoc.VerificationMethod {
 		switch k := vm.VerificationMaterial.(type) {
 		case *VerificationMethod_BlockchainAccountID:
```

Comment:
did you find something else that was problematic?

Reply:
Sorry forgot to push my latest change, ready now! :smile: